docs/operate/get-started/other-hardware/micro-module.md

```diff
@@ -204,158 +29,96 @@ To create a new module compatible with the Micro-RDK, follow these steps.
 
    Select `templates/module` when prompted, give the module a name of your choice, and answer any additional prompts.
 
-1. Change directories into the generated tree:
-
-   ```sh { class="command-line" data-prompt="$"}
-   cd <your-path-to/your-module-directory>
-   ```
+   The CLI automatically initializes a git repository in the generated module directory.
 
-1. If you wish to use version control for the module, this is the best time to initialize a git repository and commit all the generated files.
-   There are no secrets in a newly generated module repository:
+1. Navigate into the generated module directory:
 
    ```sh { class="command-line" data-prompt="$"}
-   git add .
-   git commit -m "initial commit"
+   cd <path-to/your-module-directory>
    ```
 
-1. Develop the module by defining `structs` which implement the necessary `traits` and adding tests and registration hooks for them, per the walkthrough.
-
-1. To consume the module, follow the [Build custom firmware](#build-custom-firmware) workflow in a different directory, and register your module in the `dependencies` section of the project's `Cargo.toml` file, then build and flash the project.
-   The module will now be available for use by adding it to your machine configuration on the [Viam app](https://app.viam.com).
-
-For further details on Micro-RDK development, including credentials management and developer productivity suggestions, please see the [development technical notes page on GitHub](https://github.com/viamrobotics/micro-rdk/blob/main/DEVELOPMENT.md).
-
-## Over-the-air updates
+1. Develop the module by defining structs which implement the necessary traits and adding tests and registration hooks for them.
+   The traits you need to implement are determined by the API you chose to implement.
 
-{{% hiddencontent %}}
-Over-the-air updates are available for `viam-server` and the Micro-RDK. For information about `viam-server` see [Deploy software packages to machines](/manage/software/deploy-software/).
-The following information covers the Micro-RDK.
-{{% /hiddencontent %}}
+   For example, to implement the sensor API, you need to implement `Readings`, `SensorT<f64>` and `Status` traits.
 
-To remotely update the firmware on your microcontroller without a physical connection to the device, add the OTA (over-the-air) service to your microcontroller's configuration in the [Viam app](https://app.viam.com).
-Use **JSON** mode to add the service as in the template below, then configure the URL from which to fetch new firmware, and the version name.
+   {{< expand "Example implementation" >}}
+   Here is an example <file>src/lib.rs</file> file that implements the sensor API to return random numbers:
```

Comment:
```suggestion
   Example that returns random numbers:
```

To reduce words, since we say "to implement the sensor API" just above

Reply:
```suggestion
   The following example <file>src/lib.rs</file> file implements the sensor API to return random numbers:
```
Hmm I do want it to be a complete sentence, and I think the context of which file it is can be helpful. I also think even with the preceding sentence, it's not actually clear that this example implements the sensor API.

Reply:
Sounds good